### PR TITLE
Add contact page content

### LINKED ISSUE #29: Contact page
With content

### PR DESCRIPTION
**Pull request by triszt4n**
Closes #29

### CODE REVIEW

File: src/pages/contact.tsx [hunk 1]
```diff
@@ -7,7 +7,97 @@ const ContactPage: React.FC<PageProps> = () => {
...
+          <div className="row align-items-end">
+            <div className="col-lg-7">
+              <h2 className="mb-0">Kapcsolat</h2>
+              <p>Elérhetőség, térkép és fényképek</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className="custom-breadcrumbs border-bottom">
```
Comment (bbucsy) [inline src/pages/contact.tsx:22]: There is a typo, should be  `custom-breadcrumns` OR should fix typo in *_site-blocks.scss*

File: src/pages/contact.tsx [hunk 2]
```diff
@@ -7,7 +7,97 @@ const ContactPage: React.FC<PageProps> = () => {
...
+        <div className="container">
+          <Link to="/">Kezdőlap</Link>
+          <span className="mx-3 icon-keyboard_arrow_right" />
```
Comment (bbucsy) [inline src/pages/contact.tsx]: on Firefox ( windows ) `icon-keyboard_arrow_right` doesn't seem to work.

**Review by bbucsy — CHANGES_REQUESTED** [review]
There are some small bugs 🕷️

File: src/pages/contact.tsx (hunk 1 above)
Comment (triszt4n) [reply src/pages/contact.tsx:22]: Oh jesus, so that was the problem! thx

File: src/pages/contact.tsx (hunk 2 above)
Comment (triszt4n) [reply src/pages/contact.tsx]: Yep these are silly little design problems that I will repair in an other PR but right now the content was priority not the design

**Review by bbucsy — APPROVED** [review]

**Pull request merged**